gtsam/nonlinear/tests/testValues.cpp

```diff
@@ -175,10 +175,13 @@ TEST(Values, basic_functions)
 {
   Values values;
   const Values& values_c = values;
-  values.insert(2, Vector3());
-  values.insert(4, Vector3());
-  values.insert(6, Matrix23());
-  values.insert(8, Matrix23());
+  Matrix23 M1, M2;
+  M1 << 0, 0, 0, 0, 0, 0;
+  M2 << 0, 0, 0, 0, 0, 0;
+  values.insert(2, Vector3(0, 0, 0));
+  values.insert(4, Vector3(0, 0, 0));
+  values.insert(6, M1);
```

Comment:
How about Z_2x3 ?

Reply:
`Z_2x3` isn't defined but I see what you mean.